src/TheGuardianIn2020Banner/index.stories.tsx

```diff
@@ -31,17 +31,18 @@ export const defaultStory = (): ReactElement => {
     return (
         <StorybookWrapper>
             <BrazeMessageComponent
-                componentName={componentName}
+                candidates={[
```

Comment:
I don't think this is the interface to `BrazeMessageComponent` any more right?

Reply:
Oh, I think this is what's causing the diff in Chromatic. It looks like this story rendering correctly because we're passing the props in the wrong way here.